### PR TITLE
Adding event on disabled items

### PR DESCRIPTION
**Pull request by gnassro**
Pull request for this feature [https://github.com/Frezyx/group_button/issues/58](https://github.com/Frezyx/group_button/issues/58)

### CODE REVIEW

File: lib/src/group_button_base.dart [hunk 1]
```diff
@@ -80,6 +82,7 @@ class GroupButton extends StatelessWidget {
         disabledButtons: disabledButtons,
         selectedButton: selectedButton,
         onSelected: (index, _) => onSelected(index),
+        onDisablePressed: (index,) => onDisablePressed(index),
```
Comment (Frezyx) [inline lib/src/group_button_base.dart:85]: unnecessary comma

File: lib/src/group_custom_button.dart [hunk 2]
```diff
@@ -72,24 +74,50 @@ class GroupCustomButton extends StatelessWidget {
...
+              onPressed: onPressed,
+              style: ElevatedButton.styleFrom(
+                elevation: elevation ?? 0.0,
+                primary: _getBackgroundColor(theme),
+                shape: _buildShape(),
+                padding:
+                    (width != null || height != null) ? EdgeInsets.zero : null,
+                alignment: (width != null || height != null) ? alignment : null,
+              ),
+              child: Padding(
```
Comment (Frezyx) [inline lib/src/group_custom_button.dart:88]: This type of widgets build is huge and need in much place to code 
Please simplify this code

**Review by Frezyx — CHANGES_REQUESTED** [review]

**Comment by gnassro** [comment]
what's wrong in this now ? somthing missing ?

**Review by Frezyx — APPROVED** [review]

**Comment by Frezyx** [comment]
All is good!
Thank you for your contribution @gnassro

**Comment by gnassro** [comment]
> All is good! Thank you for your contribution @gnassro

Thank you too for this package.
i think this pull request must linked to this [issue](https://github.com/Frezyx/group_button/issues/58)

**Pull request merged**